Validation/RecoTrack/plugins/TrackingNtuple.cc

```diff
@@ -1292,6 +1292,7 @@ class TrackingNtuple : public edm::one::EDAnalyzer<edm::one::SharedResources> {
       ph2_radL;  //http://cmslxr.fnal.gov/lxr/source/DataFormats/GeometrySurface/interface/MediumProperties.h
   std::vector<float> ph2_bbxi;
   std::vector<uint64_t> ph2_usedMask;
+  std::vector<size_t> ph2_size;
```

Comment:
to follow the phase-1 strips `str_clustSize`, 
```suggestion
  std::vector<size_t> ph2_clustSize;
```